### DISCUSSION TITLE
Vue 3 migrate modifiers by condition

### DISCUSSION THREAD

**Discussion by devopg** [vuejs/docs#1474] [Q&A]
I continues migrate to vue 3.
In vue 2 i use method by @pbastowski for add modifiers by condition for input

`<input type="text" v-model="filter" @[isOpen&&click.stop] />`

how can i do in vue 3 it ?

_I emphasize that I do not have my own handler only bool var_

**Comment by tiger-githubb** [comment]
I think the same syntax also works in Vue 3

**Comment by tiger-githubb** [comment]
I think the same syntax also works in Vue 3
